lib/proof.ts

```diff
@@ -1,9 +1,7 @@
 import { MerkleTree } from "merkletreejs";
 import keccak256 from "keccak256";
 
-// convert hex string (0xabcd123) to hash, hashing not the string but the actual value
-const hashStringAddress = (address: string): string =>
```

Comment:
Apparently `keccak256` converts hex strings to bytes internally already - https://github.com/miguelmota/keccak256/blob/master/index.js#L16